Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.8-bookworm as base
+
+RUN pip install poetry
+WORKDIR /app
+COPY pyproject.toml poetry.lock ./
+RUN poetry install --no-root --no-dev
+COPY todo_app /app/todo_app
+
+FROM base as production
+ENV FLASK_ENV=production
+ENV FLASK_APP=todo_app.app:create_app()
```

Comment:
This argument isn't actually needed, because you pass it through directly in the ENTRYPOINT